engine/swarm/key-concepts.md

```diff
@@ -10,19 +10,36 @@ orchestration features of Docker Engine 1.12.
 ## What is a swarm?
 
 The cluster management and orchestration features embedded in the Docker Engine
-are built using **SwarmKit**. Docker engines participating in a cluster are
-running in **swarm mode**. You enable swarm mode for an engine by either
-initializing a swarm or joining an existing swarm.
-
-A **swarm** is a cluster of Docker engines, or _nodes_, where you deploy
-[services](key-concepts.md#services-and-tasks). The Docker Engine CLI and API
-include commands to manage swarm nodes (e.g., add or remove nodes), and deploy
-and orchestrate services across the swarm.
-
-When you run Docker without using swarm mode, you execute container
-commands. When you run the Docker in swarm mode, you orchestrate services. You can run swarm services and standalone containers on the same Docker instances.
-
-## What is a node?
+are built using [swarmkit](https://github.com/docker/swarmkit/). Swarmkit is a
+separate project which implements Dockr's orchestration layer and is used
+directly within Docker.
+
+A swarm consists of multiple Docker hosts which run in **swarm mode** and act as
+managers (to manage membership and delegation) and workers, which run
```

Comment:
This language is a little confusing. Act as managers and workers doesn't make it clear that nodes could have either of these two roles.